packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -54,7 +54,7 @@ export const InboundListView: FC = () => {
       ['invoiceNumber', { maxWidth: 80 }],
       'createdDatetime',
       'allocatedDatetime',
-      ['comment', { maxWidth: 300 }],
+      ['comment', { width: '100%' }],
```

Comment:
doing this to squash the name column, without it, the name spreads too much on a wide screen

Reply:
Why not just put a max-width on the "name" column?

Reply:
Some of the names used by clients are really long - and many don't have comments. Basically just trying to cater for long names when they exist

and it didn't work for me - setting a `max-width` on name doesn't change anything unless I also add the `100%` width to comment